zingolib/src/wallet.rs

```diff
@@ -86,16 +86,16 @@ pub struct SendProgress {
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum Pool {
-    Sapling,
     Orchard,
+    Sapling,
```

Comment:
i typically write such an enum in the opposite order: transparent .. sapling .. orchard...